File: Dot examples/ASP Book Examples/Book Examples/Chapter23/CS/App_Code/UsersOnline.cs
using System;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace myControls
{

    /// <summary>
    /// Displays Number of Users Online
    /// </summary>
    public class UsersOnline : WebControl
    {
        protected override void RenderContents(HtmlTextWriter writer)
        {
            writer.Write(Membership.GetNumberOfUsersOnline());
        }

    }
}

File: Dot examples/ASP Book Examples/Book Examples/Chapter23/VB/App_Code/UsersOnline.vb
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls

Namespace myControls

    ''' <summary>
    ''' Displays Number of Users Online
    ''' </summary>
    Public Class UsersOnline
        Inherits WebControl

        Protected Overrides Sub RenderContents(ByVal writer As HtmlTextWriter)
            writer.Write(Membership.GetNumberOfUsersOnline())
        End Sub

    End Class
end namespace